ChainRouter.php

```diff
@@ -306,4 +306,25 @@ public function getRouteCollection()
 
         return $this->routeCollection;
     }
+    
+    /**
+     * Identify if any routers have been added
+     * 
+     * @return bool
+     */
+    public function hasRouters()
+    {
+        return !empty($this->routers);
+    }
+    
+    /**
+     * Get the number of added routers
+     * 
+     * @return integer
+     */
+    public function getNumRouters()
```

Comment:
a chain routers with zero routers indeed makes no sense, so I see it as something that one might always check. but the number of routers is only useful when debugging and even there I doubt that it will help in most cases, since you actually then care more about which routers are registered and less about how many have been registered. now maybe for your application its all people care about, but imho its not general enough and easy enough to get around not having.

so -1 on adding this method